snuba/snuba_migrations/generic_metrics/0014_distribution_add_options.py

```diff
@@ -0,0 +1,82 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    storage_set_key = StorageSetKey.GENERIC_METRICS_DISTRIBUTIONS
+
+    local_table_name = "generic_metric_distributions_raw_local"
+    dist_table_name = "generic_metric_distributions_raw_dist"
+
+    columns = [
+        (
+            Column("enable_histogram", UInt(8, MigrationModifiers(default=str("0")))),
```

Comment:
You should probably make MigrationModifiers default value accept `ints` in a separate PR first. I don't see a reason why it has to be restricted to strings only. Then once that PR is merged, you can use the int value itself over here. It seems a bit odd that a UInt8 column is being populated from a string.

Reply:
I was really confused about the `str` type hints there as well. I took a look at the underlying code for `MigrationModifiers`, looks like it's just generating SQL strings [under the hood](https://github.com/getsentry/snuba/blob/b0f7e434459fb5cad5a1b6a8ec9e460f9e8c1cd4/snuba/migrations/columns.py#L72) so the str here is technically an SQL expression
```python
def for_schema(self, content: str) -> str:
        return "{} DEFAULT {}".format(content, self.default)
```
I could make another pr that let default be some other type, but I think that type will need to support construction of arbitrary sql queries instead of `ints` and it needs to be turned into a string anyway.